code/modules/admin/verbs/freeze.dm

```diff
@@ -5,121 +5,95 @@
 //////Allows admin's to right click on any mob/mech and freeze them in place.///
 ////////////////////////////////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
-var/global/list/frozen_mob_list = list()
-/client/proc/freeze(var/mob/living/M as mob in GLOB.mob_list)
+var/global/list/frozen_atom_list = list()
```

Comment:
Can you make this into a managed global list? (Use the GLOBAL_LIST_INIT helpers)

Reply:
Good idea